mesonbuild/templates/valatemplates.py

```diff
@@ -48,30 +55,42 @@
 }}
 '''
 
-lib_vala_meson_template = '''project('{project_name}', 'vala',
+lib_vala_meson_template = '''project(
+  '{project_name}',
+  'vala',
   meson_version : '>= {meson_version}',
-  version : '{version}')
+  version : '{version}',
+)
 
 dependencies = [
-    dependency('glib-2.0'),
-    dependency('gobject-2.0'),
+  dependency('glib-2.0'),
+  dependency('gobject-2.0'),
 ]
 
 # These arguments are only used to build the shared library
 # not the executables that use the library.
-shlib = shared_library('foo', '{source_file}',
-               dependencies: dependencies,
-               install: true,
-               install_dir: [true, true, true])
-
-test_exe = executable('{test_exe_name}', '{test_source_file}', dependencies : dependencies,
-  link_with : shlib)
+shlib = shared_library(
+  'foo',
+  '{source_file}',
+  dependencies: dependencies,
+  install: true,
+  install_dir: [true, true, true],
+)
+
+test_exe = executable(
+  '{test_exe_name}',
+  '{test_source_file}',
+  dependencies : dependencies,
+  link_with : shlib,
+)
 test('{test_name}', test_exe)
 
 # Make this library usable as a Meson subproject.
 {ltoken}_dep = declare_dependency(
   include_directories: include_directories('.'),
-  link_with : shlib)
+  link_with : shlib,
```

Comment:
Sometimes we use `foo : bar` and sometimes `foo: bar`.

(I'd standardize on the former.)